### DISCUSSION TITLE
Comment scraping order question

### DISCUSSION THREAD

**Discussion by sk1pd1v1d3d** [kevinzg/facebook-scraper#816] [Q&A]
In what order are comments (and replies) scraped by default? Is it possible to scrape comments in a particular order, such as first scraping comments with the highest reaction count? Given the limit on the number of comments that can be scraped, I'm wondering if it's possible to prioritize scraping comments based on a specific variable, such as comment reaction count.

**Comment by neon-ninja** [comment]
Comments are scraped in the order they're presented on m.facebook.com. The scraper can't control the order. Replies will be given to you as a generator if you set "comments": "generator", so you can request replies based on your own logic.

**Reply by sk1pd1v1d3d** [reply]
Understood, thank you for explaining!
